sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -47,322 +44,259 @@
+        <version>4.0.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-security-keyvault-certificates</artifactId>
-        <version>4.1.8</version>
+        <version>4.2.0</version>
+      </dependency>
+      <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-security-keyvault-jca</artifactId>
+        <version>1.0.1</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-security-keyvault-keys</artifactId>
-        <version>4.2.8</version>
+        <version>4.3.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-security-keyvault-secrets</artifactId>
-        <version>4.2.8</version>
+        <version>4.3.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-blob</artifactId>
-        <version>12.11.1</version>
+        <version>12.12.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-blob-batch</artifactId>
-        <version>12.9.1</version>
+        <version>12.10.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-blob-cryptography</artifactId>
-        <version>12.11.1</version>
+        <version>12.12.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-common</artifactId>
-        <version>12.11.1</version>
+        <version>12.12.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-file-datalake</artifactId>
-        <version>12.5.1</version>
+        <version>12.6.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-file-share</artifactId>
-        <version>12.9.1</version>
+        <version>12.10.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-internal-avro</artifactId>
-        <version>12.0.4</version>
+        <version>12.0.5</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-queue</artifactId>
-        <version>12.9.1</version>
+        <version>12.10.0</version>
       </dependency>
-
-      <!-- Dependencies used by the Azure SDKs -->
       <dependency>
         <groupId>com.fasterxml.jackson</groupId>
         <artifactId>jackson-bom</artifactId>
-        <version>2.12.2</version>
+        <version>2.12.3</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-
+      <dependency>
+        <groupId>com.fasterxml.jackson.module</groupId>
```

Comment:
Let's add a rule to not include individual Jackson dependencies and just let the jackson-bom handle the versioning

Reply:
I do have that but I also check the closure. This may be a bug in the tool or is a library not getting closure from the BOM. I will dig in a bit and report back.

Reply:
Cosmos takes direct dependency on this library and it is not present in the BOM. 

How is this extra dependency any different than say netty-tcnative-boringssl-static which does not get its closure from netty BOM.